### PR TITLE
[Fix] Fixed the top page. (display data of plan and places.)

### PR DESCRIPTION
**Pull request by ykhoop**
Fixed the top page for general users.
- add the links to search page
- display data of place
- display data of places
- display images of 'how to' cards

### CODE REVIEW

File: public/images/top_howto_login.png
Comment (keanbillyfernandez) [inline public/images/top_howto_login.png:1]: Is the buttons in webpage an image? Or this is just for tutorial to login?

Comment (ykhoop) [reply public/images/top_howto_login.png:1]: It's just an image for telling how to use it.

**Review by ykhoop — COMMENTED** [review]
I commented about the image for login that keanbillyfernandez commented .

File: database/migrations/2023_06_20_123227_change_image_column_nullable_in_places_table.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('places', function (Blueprint $table) {
+            DB::statement('UPDATE `places` SET `image` = "" WHERE `image` IS NULL');
```
Comment (raymonkredo) [inline database/migrations/2023_06_20_123227_change_image_column_nullable_in_places_table.php:30]: let's transfer this part somewhere else

File: database/migrations/2023_06_20_123227_change_image_column_nullable_in_places_table.php [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('places', function (Blueprint $table) {
+            DB::table('places')->whereNull('image')->update(['image' => '']);
```
Comment (raymonkredo) [inline database/migrations/2023_06_20_123227_change_image_column_nullable_in_places_table.php:30]: just move this outside the schema

Comment (ykhoop) [reply database/migrations/2023_06_20_123227_change_image_column_nullable_in_places_table.php]: I moved the updating eloquent outside the schema

**Review by raymonkredo — APPROVED** [review]

**Pull request merged**